.github/workflows/release.yml

```diff
@@ -27,14 +28,15 @@ jobs:
         MERGE_LABELS: automerge
         MERGE_METHOD: squash
   release-to-github:
+    if: ${{ startsWith(github.head_ref, 'prerelease/') }} && ${{ github.event.label.name == 'automerge' }}
     needs:
     - merge-release-branch
     runs-on: [self-hosted, linux]
     steps:
     - name: Check out develop
       uses: actions/checkout@v2
       with:
-        ref: develop
+        ref: release/1.2.0
```

Comment:
This is more specific than I would expect here. Just wanted to check that it's intended.

Reply:
yeah need to find a proper variable for 'target branch name'...

Reply:
`${{ github.event.pull_request.base.ref }}` this will only be on this branch. Will make a proper PR to develop with this change